docs/production-deployment/cloud/nexus/getting-started.mdx

```diff
@@ -45,7 +47,7 @@ New Nexus requests will be routed to the new target Namespace, and existing Nexu
 
 1. Hide service implementations behind a Nexus Endpoint.
    1. Add Nexus Services to the same Workers as the Temporal primitives being abstracted.
-   2. Add Nexus Endpoints to the Nexus API Registry, with monolithic Namespace as the target.
```

Comment:
This is a great place to(not this line, obviously, but this area) to talk about another great Nexus feature -- the gradual adoption to better architecture outcomes over time.

Reply:
Good idea. Will add that in a future PR.